### ISSUE TITLE
Bad response when using query parameters when getting documents

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/MeiliSearch#446]
At some point, when adding  the query parameter `attributesToRetrieve` on the `GET documents` route, the parameter are corrupted and changed.

The first returned document on the two following requests are different from each other (note: i'm using the moviedataset): 
```bash
curl \
  -X GET 'http://localhost:8082/indexes/test_uid/documents?offset=1&attributesToRetrieve=id' | jq
``` 
has as 3 first results:
_for clarity I only took the first 3 but there are 20 documents in the response_

```json
[
  {
    "id": "25087"
  },
  {
    "id": "81003"
  },
  {
    "id": "31804"
  }
]
```

while this requests: 
```bash
curl \
  -X GET 'http://localhost:8082/indexes/test_uid/documents?offset=1&attributesToRetrieve=id&attributesToRetrieve=title' | jq
```

gives back the 3 following : 
```json
[
  {
    "id": "13342",
    "title": "Fast Times at Ridgemont High",
    "poster": "https://image.tmdb.org/t/p/w1280/9y5rSeO0xH3m5oRJmhBusDkiS0j.jpg",
    "overview": "Follows a group of high school students growing up in southern California, based on the real-life adventures chronicled by Cameron Crowe. Stacy Hamilton and Mark Ratner are looking for a love interest, and are helped along by their older classmates, Linda Barrett and Mike Damone, respectively. The center of the film is held by Jeff Spicoli, a perpetually stoned surfer dude who faces off with the resolute Mr. Hand, who is convinced that everyone is on dope.",
    "release_date": 398048400
  },
  {
    "id": "25087",
    "title": "Bloodsport II",
    "poster": "https://image.tmdb.org/t/p/w1280/xVfSGAbOK4FucTwkYrXJjeBFqv4.jpg",
    "overview": "After thief Alex Cardo gets caught while stealing an ancient katana in East Asia, he soon finds himself imprisoned and beaten up by the crowd there. One of the guards, Demon, feels upset by Alex appearance and tortures him as often as he gets the opportunity. Alex finds a friend and mentor in the jailhouse, Master Sun, who teaches him a superior fighting style called Iron Hand. When a 'best of the best kumite' is to take place, Demon gets an invitation. Now Master Sun and Alex need to find a way to let Alex take part in the kumite too.",
    "release_date": 825638400
  },
  {
    "id": "81003",
    "title": "Kung Fu Panda: Secrets of the Masters",
    "poster": "https://image.tmdb.org/t/p/w1280/kU8szr8xUWdMkVXifAw9r5tsuOT.jpg",
    "overview": "Po and the Furious Five uncover the legend of three of kung fu's greatest heroes: Master Thundering Rhino, Master Storming Ox, and Master Croc.",
    "release_date": 1323648000
  }
]
```

In conclusion: 

When `attributesToRetrieve` has more than one attribute to retrieve, **all parameters are ignored and all attributes are present in the returned documents as well as the offset is back to his default value.**

**Comment by bidoubiwa** [comment]
This is linked with #445. attributesToRetrieve should be formatted `attributesToRetrieve=title,id` and not `attributesToRetrieve=title&attributesToRetrieve=id`.

MeiliSearch ignores badly formatted query parameters and takes its default value in that case.
